### DISCUSSION TITLE
Welcome to HKHacking Discussions!

### DISCUSSION THREAD

**Discussion by coggy9** [coggy9/HKHacking#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wearethedeckers** [comment]
Just writing to say I'm grateful for the work you are doing on this.  I joined github just to comment and follow your work.

**Reply by rbeesley** [reply]
Welcome. If you have questions, be sure to ask.

**Reply by Stedmister** [reply]
Will do, if i can help let me know.

**Comment by jblanc88** [comment]
I would like a small request of being credited for providing the Invoke software if that's possible please. From reddit i'd be /u/theace69

**Reply by coggy9** [reply]
Sorry, I completely forgot. IIRC Harman officially released it by the time I made this repo. I've fixed this in the release for the original Cortana image.
